php/commands/taxonomy.php

```diff
@@ -81,6 +81,59 @@ public function list_( $args, $assoc_args ) {
 		$formatter->display_items( $taxonomies );
 	}
 
+	/**
+	 * Get a taxonomy
+	 *
+	 * ## OPTIONS
+	 *
+	 * <taxonomy>
+	 * : Taxonomy slug
+	 *
+	 * [--field=<field>]
+	 * : Instead of returning the whole taxonomy, returns the value of a single field.
+	 *
+	 * [--fields=<fields>]
+	 * : Limit the output to specific fields. Defaults to all fields.
+	 *
+	 * [--format=<format>]
+	 * : Accepted values: table, json, csv. Default: table
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     wp taxonomy get post_tag --format=json
+	 */
+	public function get( $args, $assoc_args ) {
+		$taxonomy = get_taxonomy( $args[0] );
+
+		if ( ! $taxonomy ) {
+			WP_CLI::error( "Taxonomy {$args[0]} doesn't exist." );
+		}
+
+		if ( empty( $assoc_args['fields'] ) ) {
+			$default_fields = array_merge( $this->fields, array(
+				'labels',
+				'cap'
+			) );
+
+			$assoc_args['fields'] = $default_fields;
+		}
+
+		$data = array(
+			'name'          => $taxonomy->name,
+			'label'         => $taxonomy->label,
+			'description'   => $taxonomy->description,
+			'object_type'   => $taxonomy->object_type,
+			'show_tagcloud' => $taxonomy->show_tagcloud,
+			'hierarchical'  => $taxonomy->hierarchical,
+			'public'        => $taxonomy->public,
+			'labels'        => $taxonomy->labels,
+			'cap'           => $taxonomy->cap,
+		);
```

Comment:
@danielbachhuber `get_taxonomy()` returns fields in a seemingly random order (labels, description, public, hierarchical, show_tagcloud, cap, name, object_type and label) so I created this `$data` variable to organize them.